pkg/stdlib/html/scroll_xy.go

```diff
@@ -41,5 +42,17 @@ func ScrollXY(ctx context.Context, args ...core.Value) (core.Value, error) {
 	x := values.ToFloat(args[1])
 	y := values.ToFloat(args[2])
 
-	return values.None, doc.ScrollByXY(ctx, x, y)
+	var opts drivers.ScrollOptions
+
+	if len(args) > 3 {
+		opts, err = toScrollOptions(args[3])
+
+		if err != nil {
+			return values.None, err
+		}
+	} else {
```

Comment:
`opts` already is `drivers.ScrollOptions{}` before `if len(args) > 3`. Therefore, this branch can be removed.